src/Security/Voter/AuthoredVoter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace AppBundle\Security\Voter;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\AuthoredInterface;
+
+class AuthoredVoter extends AbstractAdherentVoter
+{
+    protected function doVoteOnAttribute(string $attribute, Adherent $adherent, $subject): bool
+    {
+        /** @var AuthoredInterface $subject */
+        return $subject->getAuthor()->equals($adherent);
+    }
+
+    protected function supports($attribute, $subject)
+    {
+        return 'IS_AUTHOR_OF' === $attribute && $subject instanceof AuthoredInterface;
```

Comment:
What about creating an `AuthorPermissions` with an `IS_AUTHOR_OF` const ?

Reply:
I have added `AuthorVoter::PERMISSION`, is it ok for you?

Reply:
Okay for me :ok_hand: